CONTRIBUTING.md

```diff
@@ -1,34 +1,48 @@
+### How to contribute with design
 
-You can also keep in touch with [progress](https://github.com/orgs/WordPress/projects/3) and the latest updates to the project with our [WordPress contributor group](https://make.wordpress.org/openverse/).
+If you'd like to contribute to the design or front-end, feel free to propose a solution to an existing problem labeled with [Needs Design](https://github.com/WordPress/openverse-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22needs+design%22), or share an idea if you think it meets Openverse's goals.
+
+The [WordPress Design team](http://make.wordpress.org/design/) uses [Figma](https://www.figma.com/) to collaborate and share work for all WordPress projects. If you are not familiar with designing for WordPress, please carefully read the [design handbook](https://make.wordpress.org/design/handbook/). Once you have a [WordPress Slack](https://make.wordpress.org/chat/) account, join the [#design channel](http://wordpress.slack.com/messages/design/) and ask the team to set you up with a free Figma account.
+
+This will give you access to [all projects and files](https://www.figma.com/files/team/642003996626140596/WordPress.org?fuid=968867265893371002) used in WordPress.
+
+Before designing a proposal, browse the [Design Library](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=0%3A1) file to understand how Openverse has been built, and take a look at the [created work](https://www.figma.com/files/project/31962071/Openverse?fuid=968867265893371002) to get a glance at how design ideas are made. The design onboarding section in the design library file might miss documentation as it is always under construction, yet ask on [#design channel](http://wordpress.slack.com/messages/design/) for clarity.
 
-## 📖 Translations
+Once you are done and ready to share your idea, create a design issue and fill the content structure template. Please be as specific and concise as possible and feel free to add mockups, prototypes, videos, sketches, and anything that makes your idea easy to understand.
 
-This document mostly covers code and documentation-based contributions. However, you may also contribute by lending a hand in translating Openverse.
+After creating the issue, this will be labeled with `aspect: design`. The [current issues](https://github.com/WordPress/openverse-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%96%BC%EF%B8%8F+aspect%3A+design%22) can also guide you on how to describe your solution and how the discussion evolves before starting the implementation.
```

Comment:
```suggestion
After creating the issue, it will be labeled with `aspect: design`. Please reference [existing design issues](https://github.com/WordPress/openverse-frontend/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%96%BC%EF%B8%8F+aspect%3A+design%22) as a guide for how to describe your solution and to understand how the discussion evolves before implementation begins.
```